webpack.config.base.js

```diff
@@ -17,7 +19,28 @@ const config = {
             test: /(\.(png|jpg)$)/,
             loader: 'url-loader',
         }]
-    }
+    },
+    plugins: [
+        new FaviconsWebpackPlugin({
+            logo: 'assets/images/logo.png',
+            persistentCache: true,
+            inject: true,
+            background: '#fff',
+            title: 'CSS Quick Draw',
```

Comment:
Quick Draw
QuickDraw
Quickdraw

which is right? 🤔

Reply:
The English word is "quick draw", but we style it as "QuickDraw" (I guess) 🦅 👁